.github/workflows/wordpress.yml

```
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
         php: [ 5.6, 7.2 ]
-        wordpress: [ '4.9', '5.0', '5.1', '5.2', '5.3', '5.4', '5.5', latest ]
+        wordpress: [ 4.9, 5.6, latest ]
```

We can probably just do:
```suggestion
        wordpress: [ 4.9, latest ]
```

---

Ah, sure, because `latest` would cover the 5* range 👍🏻

---

Resolved via 7d767ec